README.md

```diff
@@ -82,15 +82,18 @@ from splink.duckdb.linker import DuckDBLinker
 import splink.duckdb.comparison_library as cl
 import splink.duckdb.comparison_template_library as ctl
 import splink.duckdb.blocking_rule_library as brl
+from splink.duckdb.blocking_rule_library import (
+    exact_match_rule, block_on_columns
```

Comment:
Should that just be 'block_on'?

Reply:
Sorry, I haven't re-tweaked the README yet. I can't do it in VSCode, so I've left it to last.

Reply:
You're prob already aware, but in Vs code you can use 'save without formatting' from the command palette, maybe that does help